Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -11,7 +11,13 @@
 #ifndef ALICEO2_TRD_DIGITIZER_H_
 #define ALICEO2_TRD_DIGITIZER_H_
 
+#include "TRDBase/TRDGeometry.h"
+#include "TRDBase/TRDCommonParam.h" // For kNdet & el. diffusion
+#include "TRDBase/TRDSimParam.h"
 #include "TRDBase/Digit.h"
+#include "TRDBase/TRDArraySignal.h"
+#include "TRDBase/PadResponse.h"
+
```

Comment:
Do you need all these includes? Can't you forward declare them.
As far as I can see, they are not used in the header file.

Reply:
@davidrohr, I am using forward declarations now. Thanks for the advice.